kubectl-rook-ceph.sh

```diff
@@ -226,14 +227,202 @@ function path_cm_rook_ceph_operator_config() {
+  read CONTINUE_SCRIPT
+  if [ "$CONTINUE_SCRIPT" != "yes" ]; then
+    exit 1
+  fi
+
+  # scale the operator deployment down
+  KUBECTL_NS_CLUSTER scale deployment rook-ceph-operator --replicas=0
+
+  # scale down all the mon pods
+  KUBECTL_NS_CLUSTER scale deployment -l app=rook-ceph-mon --replicas=0
+
+  # wait for the operator and mons to all stop
+  echo -e "\nWaiting for operator and mon pods to stop"
+  KUBECTL_NS_CLUSTER wait pod -l app=rook-ceph-operator --for=delete --timeout=90s
+  KUBECTL_NS_CLUSTER wait pod -l app=rook-ceph-mon --for=delete --timeout=90s
+
+  # start the mon debug pod
+  run_start_debug rook-ceph-mon-$good_mon
+
+  wait_for_pod_of_deployment_to_be_running "rook-ceph-mon-$good_mon-debug"
+
+  echo -e "\nStarted debug pod, resetting the mon quorum in the debug pod"
+
+  # run some ceph commands in the mon debug pod to reset quorum
+  set +eu
+  echo -e "\nExtracting the monmap"
+  KUBECTL_NS_OPERATOR exec deploy/rook-ceph-mon-$good_mon-debug -c mon -- ceph-mon \
+    --fsid=$ceph_fsid \
+    --keyring=/etc/ceph/keyring-store/keyring \
+    --log-to-stderr=true \
+    --err-to-stderr=true \
+    --mon-cluster-log-to-stderr=true \
+    --log-stderr-prefix=debug \
+    --default-log-to-file=false \
+    --default-mon-cluster-log-to-file=false \
+    --mon-host=$ROOK_CEPH_MON_HOST \
+    --mon-initial-members=$ROOK_CEPH_MON_INITIAL_MEMBERS \
+    --id=$good_mon \
+    --foreground \
+    --public-addr=$good_mon_public_ip \
+    --setuser-match-path=/var/lib/ceph/mon/ceph-$good_mon/store.db \
+    --public-bind-addr=$ROOK_POD_IP \
+    --extract-monmap=$monmap_path
+
+  echo -e "\nPrinting monmap"; \
+  KUBECTL_NS_OPERATOR exec deploy/rook-ceph-mon-$good_mon-debug -c mon -- monmaptool --print $monmap_path
+
+  # remove all the mons except the good one
+  for bad_mon in "${bad_mons[@]}"
+  do
+    echo -e "\nRemoving mon $bad_mon"
+    KUBECTL_NS_OPERATOR exec deploy/rook-ceph-mon-$good_mon-debug -c mon -- monmaptool $monmap_path --rm $bad_mon
+  done
+
+  echo -e "\rInjecting the monmap"
+  KUBECTL_NS_OPERATOR exec deploy/rook-ceph-mon-$good_mon-debug -c mon -- ceph-mon \
+    --fsid=$ceph_fsid \
+    --keyring=/etc/ceph/keyring-store/keyring \
+    --log-to-stderr=true \
+    --err-to-stderr=true \
+    --mon-cluster-log-to-stderr=true \
+    --log-stderr-prefix=debug \
+    --default-log-to-file=false \
+    --default-mon-cluster-log-to-file=false \
+    --mon-host=$ROOK_CEPH_MON_HOST \
+    --mon-initial-members=$ROOK_CEPH_MON_INITIAL_MEMBERS \
+    --id=$good_mon \
+    --foreground \
+    --public-addr=$good_mon_public_ip \
+    --setuser-match-path=/var/lib/ceph/mon/ceph-$good_mon/store.db \
+    --public-bind-addr=$ROOK_POD_IP \
+    --inject-monmap=$monmap_path
+  echo -e "\nFinished updating the monmap!"
+  set -eu
+
+  echo -e "\nPrinting final monmap"
+  KUBECTL_NS_OPERATOR exec deploy/rook-ceph-mon-$good_mon-debug -c mon -- monmaptool --print $monmap_path
```

Comment:
```suggestion
  KUBECTL_NS_CLUSTER exec deploy/rook-ceph-mon-$good_mon-debug -c mon -- monmaptool --print $monmap_path
```
`KUBECTL_NS_OPERATOR` is this right or it will be `KUBECTL_NS_CLUSTER`